### PR TITLE
Move integration/common from marquez

### PR DESCRIPTION
**Pull request by mobuchowski**
Besides renaming, had to do slight moving of directories to conform to [PEP420](https://www.python.org/dev/peps/pep-0420/) with openlineage-python and openlineage-integration-common. Basically, moving code out of top-level directory `openlineage` to `openlineage.client` and `openlineage.common`.

Signed-off-by: Maciej Obuchowski <maciej.obuchowski@getindata.com>

### CODE REVIEW

**Comment by julienledem** [comment]
per the Apache License, please add a NOTICE file that mentions that this code was originally from the Marquez Project: https://www.apache.org/licenses/LICENSE-2.0.html#redistribution

**Comment by julienledem** [comment]
Otherwise this looks good to me.

**Comment by julienledem** [comment]
Something along the lines of "This project includes code from the Marquez Project, part of the LFAI & Data Foundation"

**Comment by mobuchowski** [comment]
Added.

**Review by wslulciuc — CHANGES_REQUESTED** [review]
To be consistent with the existing folder naming for OpenLineage, I would make `/integrations` singular (i.e. `/integration`) to follow the same naming convention of our clients: `/client`

File: .circleci/config.yml [hunk 1]
```diff
@@ -83,3 +122,6 @@ workflows:
       - release-java:
           context: release
           <<: *only-on-release
+      - release-integration-python:
```
Comment (wslulciuc) [inline .circleci/config.yml:125]: Why not combine the python release of `release-client-python` and `release-integration-python` into a single job named `release-python`?

Comment (mobuchowski) [reply .circleci/config.yml]: Done

**Review by wslulciuc — APPROVED** [review]
LGTM 💯 🥇

**Pull request merged**